### PR TITLE
add register button in events section

### PR DESCRIPTION
**Pull request by ashish7515**

### CODE REVIEW

**Review by Ankur-Agrawal-ece20 — CHANGES_REQUESTED** [review]

File: src/Mashal/components/MashalEvent/mashalEvent.css
Comment (RainaJain5) [inline src/Mashal/components/MashalEvent/mashalEvent.css:1]: Reduce space between events heading and event cards

File: src/Udyam/components/Events/events.js [hunk 1]
```diff
@@ -221,6 +221,15 @@ function Events() {
                 <>PROBLEM STATEMENT</>
               )}
             </div>
+            <div className="register-link">
+              {eventData.registerLink ? (
+                <a href="https://eesiitbhu.in/dashboard#events-register" target="_blank" rel="noreferrer">
+                  REGISTER
```
Comment (RainaJain5) [inline src/Udyam/components/Events/events.js:227]: 1. Define eventData.registerLink in the data array.
2. Remove target="_blank" rel="noreferrer" from this button

**Review by RainaJain5 — CHANGES_REQUESTED** [review]

File: src/Udyam/components/Events/events.js [hunk 2]
```diff
@@ -21,50 +21,68 @@ const data = [
...
     psLink: 'https://drive.google.com/drive/folders/1cR8pelM4daqMg530qPYvZqKcMk8fOx88?usp=share_link',
     submissionLink: ''
   },
   {
     title: 'FUNCKIT',
     description: 'Test your problem-solving abilities through low-level programming and building gate-level circuitry to optimize the logic and hardware and decrease execution time.',
     psLink: 'https://drive.google.com/file/d/1PAd6g6Qw9HTc3o9usjeKzlSbVzTqbZ0P/view?usp=share_link',
-    submissionLink: ''
+    submissionLink: '',
+    registerLink: 'https://eesiitbhu.in/dashboard#events-register'
```
Comment (RainaJain5) [inline src/Udyam/components/Events/events.js:85]: psLink and submissionLink needs to be defined only once in every event array. 
Add *,* after giving link of every variable

File: src/Udyam/components/Events/events.js [hunk 3]
```diff
@@ -235,6 +253,7 @@ function Events() {
                 <>PROBLEM STATEMENT</>
               )}
             </div>
+            <div className="register-link">{eventData.registerLink ? <a href={eventData.registerLink}>REGISTER</a> : <>REGISTER</>}</div>
```
Comment (RainaJain5) [inline src/Udyam/components/Events/events.js:256]: Redirect to sign in page if the user is not logged in on clicking this button.

**Review by RainaJain5 — CHANGES_REQUESTED** [review]
Remove merge conflicts.

File: src/Udyam/components/Events/events.js (hunk 2 above)
Comment (ashish7515) [reply src/Udyam/components/Events/events.js]: Done

**Review by RainaJain5 — APPROVED** [review]

**Pull request merged**